backend/account/src/main/java/com/htec/account/service/UserService.java

```diff
@@ -168,6 +168,19 @@ public UserResponseModel updateUser(UserDTO userDTO) {
         return UserMapper.INSTANCE.userEntityToUserResponseModel(user);
     }
 
+    public UserResponseModel updateUserProfilePicture(UpdateUserPhotoByIdRequestModel updateUserPhotoByIdRequestModel) {
+
+        UserEntity user = userRepository.findById(updateUserPhotoByIdRequestModel.getId())
+                .orElseThrow(ExceptionSupplier.recordNotFoundWithId);
+
+        if (updateUserPhotoByIdRequestModel.getPictureName() != null)
```

Comment:
update only if photo exists, and not always

Reply:
Resolved with
 if (files == null || files.get("image") == null)
            throw ExceptionSupplier.couldNotSaveImage.get();
in FileService.java